pkg/virt-operator/creation/rbac/controller.go

```diff
@@ -56,6 +57,21 @@ func CreateControllerRBAC(clientset kubecli.KubevirtClient, kv *virtv1.KubeVirt,
 		log.Log.V(4).Infof("serviceaccount %v already exists", sa.GetName())
 	}
 
+	pspcli := clientset.PolicyV1beta1()
```

Comment:
ha, i don't actually think this function is called anywhere now. It doesn't matter for the sake of this PR or not.

I was supposed to remove functions like "CreateControllerRBAC" back when a large refactoring was done.